diff -up zxing-cpp-master/core/CMakeLists.txt.1~ zxing-cpp-master/core/CMakeLists.txt
--- zxing-cpp-master/core/CMakeLists.txt.1~	2018-10-17 23:42:52.000000000 +0200
+++ zxing-cpp-master/core/CMakeLists.txt	2018-11-26 17:44:47.396143591 +0100
@@ -472,13 +472,14 @@ target_compile_options (ZXingCore
 )
 
 add_library(ZXing::Core ALIAS ZXingCore)
+set_target_properties(ZXingCore PROPERTIES VERSION 0 SOVERSION "0.0.0")
 set_target_properties(ZXingCore PROPERTIES EXPORT_NAME Core)
 
-set(CMAKECONFIG_INSTALL_DIR "lib/cmake/ZXing")
+set(CMAKECONFIG_INSTALL_DIR "${LIB_INSTALL_DIR}/cmake/ZXing")
 install(TARGETS ZXingCore EXPORT ZXingTargets
-    LIBRARY DESTINATION lib
+    LIBRARY DESTINATION ${LIB_INSTALL_DIR}
     RUNTIME DESTINATION bin
-    ARCHIVE DESTINATION lib
+    ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
     INCLUDES DESTINATION include
 )
 install(EXPORT ZXingTargets DESTINATION ${CMAKECONFIG_INSTALL_DIR} NAMESPACE ZXing::)
